backend/incident_manager/models/incident.py

```diff
@@ -11,16 +14,30 @@ class Incident(models.Model):
         on_delete=models.SET_NULL,
         related_name="incidents",
     )
+    thana = models.ForeignKey(
+        Thana,
+        null=True,
+        default=None,
+        on_delete=models.SET_NULL,
+        related_name="incidents",
+    )
     location = models.PointField(
         geography=True, null=True, default=None
     )  # Using GeoDjango for geo fields
     description = models.TextField()
     additional_info = models.JSONField(
         default=dict
     )  # Set default to an empty dictionary
-    status = models.CharField(max_length=50)
+    status = models.CharField(
```

Comment:
Incident model will have two types of status- 
- validation_status: ("pending", "validated", "rejected")
- task_status:  ("opened", "closed", "postponed")
Task status will indicate that if any volunteer is acted on this to rescue or not, after rescue the status will be "closed", if it's not possible to act then it'll be "postponed". kindly add the properties

Reply:
Done.